js/foundation.drilldown.js

```diff
@@ -85,7 +85,11 @@ class Drilldown {
       var $menu = $(this),
           $back = $menu.find('.js-drilldown-back');
       if(!$back.length){
-        $menu.prepend(_this.options.backButton);
+        if(_this.options.backButtonPosition == 'bottom'){
+          $menu.append(_this.options.backButton);
+        } else {
+          $menu.prepend(_this.options.backButton);
+        }
```

Comment:
Would adding such a statement force users to specify a position? My concern was that could cause any current usage of the drilldown menu to break on update, though I suppost I could check for "top", "bottom" or unset and only warn if there *is* a value but it's not top or bottom. Thoughts?